CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ be found.
 
 ## 1.13.0 (2016-12-08)
 
+**IMPORTANT**: In Docker 1.13 managed plugin api changed from the experimental
+version introduced in Docker 1.12. Plugins installed with Docker 1.12 should
+therefore be **uninstalled** _before_ upgrading to Docker 1.13. You can
+uninstall plugins using the `docker plugin rm` command.
+
+If you have already upgraded to Docker 1.13 without uninstalling plugins, and
+get this error message;
```

Comment:
We need to add details on when this error is encountered.
i.e `get this error message during docker daemon startup`